scripts/releasing/patch-trigger.js

```diff
@@ -101,45 +101,32 @@ async function main() {
     );
   }
 
-  const releaseRef = `release/${version}`;
-  const workflowId =
-    context.eventName === 'pull_request'
-      ? 'release-patch-3-release.yml'
-      : process.env.WORKFLOW_ID || 'release-patch-3-release.yml';
-
-  console.log(`Detected channel: ${channel}, version: ${version}`);
-  console.log(`Release ref: ${releaseRef}`);
-  console.log(`Workflow ID: ${workflowId}`);
-  console.log(`Dry run: ${isDryRun}`);
-
-  if (testMode) {
-    console.log('\n🧪 TEST MODE - No API calls will be made');
-    console.log('\n📋 Parsed Results:');
-    console.log(`  - Branch: ${headRef}`);
-    console.log(
-      `  - Channel: ${channel} → npm tag: ${channel === 'stable' ? 'latest' : 'preview'}`,
-    );
-    console.log(`  - Version: ${version}`);
-    console.log(`  - Release ref: ${releaseRef}`);
-    console.log(`  - Workflow: ${workflowId}`);
-    console.log(`  - Dry run: ${isDryRun}`);
-    console.log('\n✅ Channel detection logic working correctly!');
-    return;
-  }
-
   // Try to find the original PR that requested this patch
   let originalPr = null;
   if (!testMode) {
     try {
       console.log('Looking for original PR using search...');
       const { execFileSync } = await import('node:child_process');
 
+      // Split search string into searchArgs to prevent triple escaping on the quoted filters
+      const searchArgs =
```

Comment:
I've lost the thread on this but can't we just parse this info from the branch names? That might be more deterministic than seraching?

Reply:
So we've included the original PR number on the **release** branches, but not the **hotfix** branch, which is the input passed in here.
Release branch ex - release/v0.7.0-preview.1-pr-1234
Hotfix branch ex- hotfix/v0.7.0-preview.1/preview/cherry-pick-4c5ab80

Hotfix branch does not have the original pr info. Unless we modify our workflow to pass that along as well. Open to doing that if you think it's a better idea.